### ISSUE TITLE
Unexpected Behavior: S3 copy() - list of dicts not properly serializes into JSON array

### ISSUE THREAD

**Issue by njdanielsen** [awslabs/aws-data-wrangler#514]
**Describe the bug**

Column contains a list of python dicts with form [{'id' : '123', 'score': '123.2' }]

My expected behavior would be that this library serializes it into a JSON array which is supported by [Redshift](https://docs.aws.amazon.com/redshift/latest/dg/json-functions.html).

Here's the error:
```
2021-01-11 14:58:33,579 | awswrangler.redshift | ERROR | Unsupported Redshift type: array<struct<id:string,score:string>>
2021-01-11 14:58:33,579 | __main__ | ERROR | Unsupported Redshift type: array<struct<id:string,score:string>>
```
**To Reproduce**
Steps to reproduce the behavior. Also add details about Python and Wrangler's version and how the library was installed.

*P.S. Don't attach files. Please, prefer add code snippets directly in the message body.*

Anywho, love this library. It's great for connecting all the pipes and saved me from writing my own version of it for internal use.

**Comment by igorborgest** [comment]
Hi @njdanielsen!

We've just added the `serialize_to_json` boolean argument (default `False`) to the function.
It is required because the `SERIALIZETOJSON` parameter is still in preview mode and is not accepted in regular clusters.
After this Redshift feature becomes GA we can make this argument `True` by default.

Do you mind to test it from our development branch  Could you give it a try?

```sh
pip uninstall awswrangler -y
pip install git+https://github.com/awslabs/aws-data-wrangler.git
```

**Comment by njdanielsen** [comment]
@igorborgest just took a look. I don't have this parameter in my cluster, but the implementation seems okay.

Thanks for the quick action!

**Comment by igorborgest** [comment]
Released on version [2.4.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/2.4.0).
